src/services/personnel.spec.ts

```diff
@@ -0,0 +1,124 @@
+        //In case this function runs prior to getting a valid id, 1146777 acts as a test id from UAT Env
+        let url = '/personnel/' + aValidPersonnelId;
+        let res: IResponse<Personnel> = await client.get<Personnel>(url);
+        expect(res.success).toBeTruthy(res.message);
+    });
+
+    it("Can read a personnel for invalid id", async () => {
+        let url: string = '/personnel/5555444';
+        let res: IResponse<Personnel> = await client.get<Personnel>(url);
+        expect(res.success).toBeFalsy(res.message);
+    });
+
+    it("Can add personnel with valid data", async () => {
+        //In case this function runs prior to getting a valid id, 1146777 acts as a test id from UAT Env
+        let url = '/personnel/' + aValidPersonnelId;
+        let resGet: IResponse<Personnel> = await client.get<Personnel>(url);
+
+        resGet.result.FirstName = "John";
+        resGet.result.LastName = "Doe";
+        let exPersonnel: Personnel[] = [resGet.result];
+
+        let resPost: IResponse<Personnel[]> = await client.post<Personnel[]>('/personnel', exPersonnel);
+        expect(resPost.success).toBeTruthy(resPost.message);
+    });
+
+    it("Can update personnel with valid data", async () => {
+        //In case this function runs prior to getting a valid id, 1146777 acts as a test id from UAT Env
+        let urlGet = '/personnel/' + aValidPersonnelId;
+        let resGet: IResponse<Personnel> = await client.get<Personnel>(urlGet);
+
+        resGet.result.Title = "Quality Analyst";
+        resGet.result.LastName = "Cena";
+        resGet.result.ROW_VERSION = "";
+
+        let exPersonnel: Personnel = resGet.result;
+        let urlPut = '/personnel/' + resGet.result.PersonnelId;
+
+        let resPut: IResponse<Personnel> = await client.put<Personnel>(urlPut, exPersonnel);
+        expect(resPut.success).toBeTruthy(resPut.message);
+    });
+
+    it("Can update personnel with invalid data", async () => {
+        //In case this function runs prior to getting a valid id, 1146777 acts as a test id from UAT Env
+        let urlGet = '/personnel/' + aValidPersonnelId;
+        let resGet: IResponse<Personnel> = await client.get<Personnel>(urlGet);
+
+        resGet.result.Notes = "A random note";
+        //Not clearing off the existing row_version
+        
+        let exPersonnel: Personnel = resGet.result;
+        let urlPut = '/personnel/' + resGet.result.PersonnelId;
+
+        let resPut: IResponse<Personnel> = await client.put<Personnel>(urlPut, exPersonnel);
+        expect(resPut.success).toBeFalsy(resPut.message);
+    });
+
+    it("Can delete a personnel with valid id", async () => {
+        //In case this function runs prior to getting a valid id, 1146777 acts as a test id from UAT Env
+        let url = '/personnel/' + aValidPersonnelId;
+        let res: IResponse<Personnel> = await client.delete<Personnel>(url);
+        expect(res.success).toBeTruthy(res.message);
+    });
+
```

Comment:
for test descriptions, the test description should match the expected outcome of the test, e.g. the below test should read "**cannot** delete a personnel with invalid id" since the expectation is that it will fail to delete the personnel record